linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_tcp_invalid_ratelimit/rule.yml

```diff
@@ -0,0 +1,58 @@
+documentation_complete: true
+
+prodtype: wrlinux1019,rhel7,rhel8
+
+title: 'Configure kernel To Rate limit sending of duplicate TCP acknowledgments'
+
+description: |-
+    Make sure that the system is configured to rate limit sending of duplicate TCP acknowledgments.
+    This measure protects against or limits effects of DoS attacks against the system.
+    Set the system to implement rate-limiting measures by adding the following line to <tt>/etc/sysctl.conf</tt> or a configuration file in the <tt>/etc/sysctl.d/</tt> directory (or modify the line to have the required value):
+    <pre>net.ipv4.tcp_invalid_ratelimit = 500</pre>
```

Comment:
This XCCDF explicitly sets the value of ``net.ipv4.tcp_invalid_ratelimit`` to ``500``.

- Must net.ipv4.tcp_invalid_ratelimit equal 500? Can a lessor or greater value be used?
- Should this be converted to a variable?

Reply:
@vojtapolasek The `sysctl` already allows for use of variables.
Check rule [sysctl_net_ipv4_conf_all_log_martians](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_conf_all_log_martians/rule.yml) and variable [sysctl_net_ipv4_conf_default_log_martians_value](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_conf_default_log_martians_value.var)